src/components/UI/Loader/Loader.css

```diff
@@ -0,0 +1,25 @@
+.loader_wrapper {
+	position: absolute;
+	width: 15rem;
+	height: 15rem;
+	top: 45%;
+	left: 50%;
+	transform: translateY(-50%) translateX(-50%);
+}
+
+.modal_container .modal {
+	border-radius: 4px;
+	border: 2px solid var(--COLOR-BLACK);
+	background-color: var(--COLOR-BG);
+	max-width: 600px;
+	padding: 1rem;
+}
+
+.modal_container {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	right: 0;
+	left: 0;
+	z-index: 999;
```

Comment:
You can create a variable for zindex stack

Reply:
ok, will do that.